plankton/adapters/__init__.py

```diff
@@ -85,78 +83,6 @@ def handle(self, cycle_delay=0.1):
         pass
 
 
-def is_adapter(obj):
-    try:
-        return issubclass(obj, Adapter) and not obj.__module__.startswith('plankton.adapters')
-    except TypeError:
-        return False
-
-
-def get_available_adapters(device_name, device_package):
```

Comment:
Yes, I've started going in that direction in the Adapter vs. Interface change that we've been discussing. I agree that there should be nothing in `__init__.py` for this module.